### PR TITLE
Add unit tests for testwick

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
The change includes couple of unit tests for testwick and different cases
covered with golang table driven approach.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37408

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: cmd/tools/testwick/testwicker_test.go [hunk 1]
```diff
@@ -0,0 +1,734 @@
...
+			},
+			expected: func(wicker *TestWicker, err error) {
+				assert.NotNil(t, err)
+				assert.Contains(t, err.Error(), "failed to add a team member")
+			},
+		},
+		{
+			description: "add team member success",
+			setup: func(wicker *TestWicker) {
+				wicker.userID = "1234"
```
Comment (Szymongib) [inline cmd/tools/testwick/testwicker_test.go:448]: This test is failing on my machine, I think we need to specify `teamID` also.
```suggestion
				wicker.teamID = "1234"
				wicker.userID = "1234"
```

**Review by Szymongib — COMMENTED** [review]
Nice tests! One is failing but that should be an easy fix.

I think we should run those as part of CI so that we can catch those things on PRs.

File: cmd/tools/testwick/testwicker_test.go (hunk 1 above)
Comment (Szymongib) [reply cmd/tools/testwick/testwicker_test.go:448]: Sorry I think mocks need to be adjusted too.

Comment (spirosoik) [reply cmd/tools/testwick/testwicker_test.go:448]: let me check

**Comment by spirosoik** [comment]
@Szymongib issues fixed and added the tests in CI too.

File: cmd/cloud/main.go [hunk 2]
```diff
@@ -15,6 +16,7 @@ var rootCmd = &cobra.Command{
 	Use:   "cloud",
 	Short: "Cloud is a tool to provision, manage, and monitor Kubernetes clusters.",
 	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Print("skata")
```
Comment (Szymongib) [inline cmd/cloud/main.go:19]: Is it some leftover from print debugging, or am I missing something? 😄

**Review by Szymongib — APPROVED** [review]
Awesome! Thanks a lot for adding CI!

**Pull request merged**